### ISSUE TITLE
Model not update immediately when remove content using long click on backspace

### ISSUE THREAD

**Issue by GrapeBaBa** [angular-ui/ui-tinymce#160]
Hi team,

I found this issue, when delete content using long click backspace, the model not update immediately. The model will update when click the mouse next time.

**Comment by deeg** [comment]
Can you please post a Plunker or Fiddle with the configuration you use re-creating the problem.

I will take a brief look to see if I see this issue in my setup as well.

**Comment by common-link-dev** [comment]
Hello,

I can confirm a problem here.

Delete the text by holding backspace until all the text is deleted. Then release the key, the model is not updated. If you then hit a letter/number or even the arrow keys, the model gets updated.

**Comment by deeg** [comment]
I will take another look into this as soon as I can. If you have any proposed fixes please feel free to post a pull request.

Also if you could post a plunker demonstrating the issue, that would be great.

Some more information would also be helpful, such as browser (with version) and library version number.

**Comment by common-link-dev** [comment]
Hello deeg,

here's the requested plunker: http://plnkr.co/edit/GjdkG2nT6WiC7Ec06FEv?p=preview

I was able to reproduce the problem in Chrome 47, Chrome 48, Firefox 42, Internet Explorer 10 mode in Internet Explorer 11. It works well in Edge mode in Internet Explorer 11.

It seems that changing the cursor position (even if the cursor is not moving) the model is updating. I was able to use the arrow, pos1, end, pgn up/down keys to update the model. But using letters, numbers, characters won't update it.
